src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,56 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 20px;
+
+  @include onTablet {
+    grid-template-columns: repeat(6, 1fr);
+    column-gap: 30px;
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+    column-gap: 30px;
+  }
+
+  &__block {
```

Comment:
Consider removing 'display: block;' from the '&__block' class, as it's not necessary. The default display of a div is 'block', so this rule is redundant.